### ISSUE TITLE
Refactor OpRegion access

### ISSUE THREAD

**Issue by avdgrinten** [qword-os/lai#25]
The I/O code in `opregion.c` needs an overhaul.

- The field flags (ByteAcc/WordAcc etc.) determine the *minimum* access width. However, accesses are still naturally aligned. Hence, the code should first round of the access size to natural alignment.
- Field offsets and sizes are *always* in bits, never in bytes.
- There is no need to special case PCI access.

**Comment by avdgrinten** [comment]
More concretely: refactor the code such that it works like this (for reads, writes need to be adjusted accordingly):
```c
[...] // Do preparation: Get BBN etc. for PCI opregions.
int width = lai_get_access_width(node); // New function to determine the access width in bytes.
int bit_progress = 0; // # of bits processed so far.
while(bit_progress < node->fld_size) {
    [...] // Perform I/O on width bytes using inb/inw/ind etc.
    [...] // Mask and shift the value that was read
    [...] // Shift and OR the value into the result
    bit_progress += /* # of bits read */
}
```

**Comment by avdgrinten** [comment]
Fixed by #54.
